_usage/command-line.md

```diff
@@ -6,14 +6,18 @@ nav_order: 1
 
 ## Using client tools to work with Babelfish
 
-Many people use SQL Server Management Studio (SSMS) on MS Windows to connect to Babelfish. 
+Most people use SQL Server Management Studio (SSMS) on Microsoft Windows to
+connect to Babelfish.
```

Comment:
Avoid subjective terms like 'most people'. Because I don't. And my friends don't either.